typescript/best-practice-web/test/specs/visual.spec.ts

```diff
@@ -0,0 +1,20 @@
+import loginPage from '../pages/login.page';
+import {LOGIN_USERS} from '../e2eConstants'
+import cartPage from '../pages/cart.page';
+import inventoryPage from '../pages/inventory.page';
+
+describe('My web app', () => {
+    it('should look correct on desktop web', () => {
+        loginPage.open();
+        browser.execute('/*@visual.init*/', 'Sauce Demo App');
+        loginPage.takeSnapshot();
+
+        loginPage.login(LOGIN_USERS.STANDARD);
+        inventoryPage.takeSnapshot();
+
+        cartPage.open();
+        cartPage.takeSnapshot();
+
```

Comment:
There is no assertion here. I know that `cartPage.takeSnapshot();` is the assertion, but maybe explain that with a comment to make it more clear for the users